src/components/Dialogs/ConnectionStatusDialog.vue

```diff
@@ -25,9 +25,28 @@ const connectionStatusColor = computed(() => {
   }
 })
 
+const geoDetails = ref<string | null>(null)
```

Comment:
I would include it in the logStore alongside the ip address to trigger only when the ip is updated instead of at each mount